GeneratorPgn.py

```diff
@@ -0,0 +1,318 @@
+import chess
+import torch.multiprocessing as mp
+import bz2
+from collections import deque
+import random
+import urllib
+import numpy as np
+
+class GeneratorPgn():
+    def __init__(self, replay_queue: mp.Queue, pgn_path):
+
+        """
+        Iterable class that generates stochastic fen and value samples by parallel running generators.
+        :param replay_queue: target queue for samples
+        :param current_line: currently used line from the data file, representing one game

```

Comment:
docstring is for function parameters and return value only, if you want to describe class variables use a comment instead